training/src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,89 @@
+package wolox.training.models;
+
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import java.time.LocalDate;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import javax.persistence.Id;
+import wolox.training.exceptions.BookAlreadyOwnedException;
+
+@Entity
+@Table(name = "users")
+public class User {
+
+    @Id
+    @GeneratedValue
+    private int Id;
+
+    @Column(nullable = false)
+    private String username;
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(nullable = false)
+    private LocalDate birthdate;
+
+    @ManyToMany(cascade = CascadeType.ALL)
+    @JoinTable(
+        name = "book_users",
+        joinColumns = @JoinColumn(name = "book_id", referencedColumnName = "id"),
+        inverseJoinColumns = @JoinColumn(name = "user_id", referencedColumnName = "id"))
+    @JsonIgnoreProperties("users")
+    private List<Book> books = new LinkedList<>();
+
+    public User() { }
+
+    public int getId() {
+        return Id;
+    }
+
+    public String getUsername() {
+        return username;
+    }
+
+    public void setUsername(String username) {
+        this.username = username;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public LocalDate getBirthdate() {
+        return birthdate;
+    }
+
+    public void setBirthdate(LocalDate birthdate) {
+        this.birthdate = birthdate;
+    }
+
+    public List<Book> getBooks() {
+        return (List<Book>) Collections.unmodifiableList(books);
+    }
+
+    public void setBooks(List<Book> books) {
+        this.books = books;
+    }
+
+    public void addBook(Book book) throws BookAlreadyOwnedException {
+        if (books.contains(book))
```

Comment:
Just a little thing: part of the good practices is using { } even if the if-statement has only one line. The main reason is [this one](https://www.quora.com/Why-would-you-use-curly-braces-in-an-if-statement)